span_customizer.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2020 The OpenZipkin Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package zipkin
+
+import "time"
+
+// SpanCustomizer allows to safely customize a span without accesing its lifecycle
+// methods
+type SpanCustomizer interface {
```

Comment:
🖐️

Reply:
name is notably missing here and needed I think https://github.com/openzipkin/brave/tree/master/instrumentation/grpc#rpc-model-mapping